client/src/App.js

```diff
@@ -1,48 +1,27 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 // import GlobalStyles from "./GlobalStyles";
 import { getQuestions } from "./api/getQuestions";
 import QuestionBubble from "./components/QuestionBubble";
 
-// function App () {
-//   const [ questions, getQuestions ] = useState(null);
-// }
+function App() {
+  const [questions, setQuestions] = useState(null);
 
-// useEffect(() => {
-//   async function getQuestions() {
+  useEffect(() => {
+    async function fetchQuestions() {
+      const allQuestions = await getQuestions();
+      setQuestions(allQuestions);
+    }
+    fetchQuestions();
+  }, []);
 
```

Comment:
Maybe you can make hook, for fetching.
It could look like this:

```
import React from "react";

export default function useFetch(url) {
    const [data, setData] = React.useState([]);

    React.useEffect(() => {
        async function fetchData() {
            const response = await fetch(url, {
                method: "GET",
                headers: {
                    "Content-Type": "application/json",
                }
            });
            
            const newData = await response.json();
            setData(newData);
        }
        fetchData();
    }, [url]);

    return data;
}

```

This hook you can reuse.

In your case:

import useFetch from 'path/to/it'

const allQuestions = useFetch("/api/questions")

Reply:
thank you for the feedback! The function has since been updated however appreciate your proposed solution!